docs/ISSUE_SPECIFICATION.md

```diff
@@ -0,0 +1,38 @@
+# issue specification
+
+- we use [lot issues] (https://github.com/openIMSDK/openKF/issues) before reporting bugs, Please make sure that has been
+  searching similar [question] (https://github.com/OpenIMSDK/OpenKF/issues), because they may have already answered or

```

Comment:
the link didn't embed

Reply:
i will add the link later, before v0.0.2

Reply:
maybe  document is not compatible， i will check it

Reply:
already fix it